fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -88,11 +88,7 @@ countries:
                         lat: Y
                         lon: X
                         time: T
-                    colormap: &raincm "[16777215 16777215 12632256 12632256 14155730
-                        [14155730 21] 12841150 [12841150 23] 10215830 [10215830 23]
-                        7262835 [7262835 22] 6931300 [6931300 23] 5944410 [5944410 23]
-                        5285200 [5285200 23] 4625990 [4625990 22] 3966780 [3966780 23]
-                        3307570 [3307570 23] 2648360 [2648360 23] 1989150]"
+                    colormap: &raincm precip
```

Comment:
We don't need the `&raincm` anchor anymore. That was to save us copy-pasting the whole colormap representation every time (each occurrence of `*raincm` below refers to this same value), but now that we're using a name rather than a colormap, we can just use the name everywhere.

Reply:
i.e. both `&raincm precip` and `*raincm` get replaced with `precip` everywhere. Likewise for the other colormaps.